### PR TITLE
Fix Tests

### PR DESCRIPTION
**Pull request by harveysanders**
- Update `fs-extra`
- Add `devcontainer.json`
- Fix `clc` usage; 
- Add `prettierrc`
- Remove `resolutions` for `graceful-fs` in package.json

### CODE REVIEW

File: controller/sessions.js [hunk 1]
```diff
@@ -2,7 +2,6 @@ const clc = require('cli-color');
 const _ = require('lodash');
 const inquirer = require('inquirer');
 const { waterfall } = require('async');
-const changeCase = require('change-case');
```
Comment (ptbarnum4) [inline controller/sessions.js:5]: TODO: Remove elsewhere and uninstall dependency

**Review by ptbarnum4 — COMMENTED** [review]
looking good so far!

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "quoteProps": "as-needed",
+  "requirePragma": false,
+  "semi": true,
+  "singleQuote": true,
+  "tabWidth": 2,
+  "trailingComma": "none",
+  "useTabs": false,
+  "embeddedLanguageFormatting": "auto",
+  "vueIndentScriptAndStyle": false,
+  "filepath": "/Users/opsparkowl/Documents/opspark/dev_shop/opspark/.eslintrc.js",
```
[the remark this answers is not in the document]
Comment (harveysanders) [reply .prettierrc:22]: Ah yeah I just grabbed thus default config out of the VSCode logs 😅. Good catch

**Review by ptbarnum4 — APPROVED** [review]
looking great!

at some point I think the test files should be renamed to (*.test.js or *.spec.js) for clarity

**Pull request merged**